datadog-agent.yaml

```
@@ -1,14 +1,18 @@
 package:
   name: datadog-agent
   version: 7.52.1
-  epoch: 2
+  epoch: 4
```

```suggestion
  epoch: 3
```

Unless this change is so large it's two whole epochs